Replace copper ingot with copper from original Minecraft.

Outfit8TSB opened the pull request:
## Description
<!-- Please explain why you are making this pull request. -->
<!-- Start writing below this line -->
on 1.17, the copper was added,
however, slimefun didnt removed it yet. causing the copper ingot(sf) still remaining.
this commit replaces Copper Ingot(SF) in favor of Copper ingot from vanilla minecraft.

all users making a new addon
should replace all usage of SlimefunItems.COPPER_INGOT to new ItemStack(Material.COPPER_INGOT).
## Proposed changes
<!-- Please explain what changes you have made to the code. -->
<!-- Start writing below this line -->
Remove SlimefunItems.COPPER_INGOT and replace with new ItemStack(Material.COPPER_INGOT)
## Related Issues (if applicable)
<!-- Please tag any Issues related to your Pull Request -->
<!-- Syntax: "Resolves #000" -->
<!-- Start writing below this line -->

## Checklist
<!-- Here is a little checklist you can follow. -->
<!-- Click on these checkboxes after you created the pull request. -->
<!-- Don't worry, these are not requirements. They only serve as guidance. -->
- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [ ] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.16.* - 1.20.*).
- [ ] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

Outfit8TSB committed:
Update SlimefunItems.java - Remove Copper Ingot in favor of Vanilla Copper Ingot. 

on 1.17, the copper was added,
however, slimefun didnt removed it yet. causing the copper ingot(sf) still remaining.
this commit removes Copper Ingot(SF) in favor of Copper ingot from vanilla minecraft.

all users used this version to make a new addon
should replace all usage of SlimefunItems.COPPER_INGOT to new ItemStack(Material.COPPER_INGOT)

Outfit8TSB committed:
Update SlimefunItemSetup.java - Replace Copper_Ingot(SF) with Copper_Ingot(MC)

on 1.17, the copper was added,
however, slimefun didnt removed it yet. causing the copper ingot(sf) still remaining.
this commit removes Copper Ingot(SF) in favor of Copper ingot from vanilla minecraft.

all users used this version to make a new addon
should replace all usage of SlimefunItems.COPPER_INGOT to new ItemStack(Material.COPPER_INGOT)

Outfit8TSB committed:
Update SlimefunItemSetup.java

ybw0014 commented:
I think this would never be considered before the recipe system rewrite (support interchangeable ingredients), but we can see what admins think.

Boomer-1 commented:
agreed ybw. there's too much impact that wasn't considered as it will affect every addon and break them all, along with every players base, sorting systems, network systems, etc., much more care has to be put into this before it would be rolled out

Outfit8TSB commented:
D'oh, i didnt reailze that. 😵
But you should consider that Copper was Added in 1.17.
but the copper from slimefun was really like existed in 1.17~1.21. Why do you even want to keep it? They are `@Deprecated`. It Must be replaced with `new Itemstack(Material.COPPER_INGOT)`.
If Slimefun copper is there, whats the point for Copper in Minecraft?
For that, i think the old `SlimefunItems.COPPER_INGOT` Should be removed,
And Must be replaced to `new ItemStack(Material.COPPER_INGOT)`.

gphg commented:
Let's ship it on Slimefun 5!

Outfit8TSB commented:
> Let's ship it on Slimefun 5!

shush! no saying about slimefun 5. this is slimefun 4.

Outfit8TSB committed:
Update SlimefunItemSetup.java - fix errors

it should fix Error:  /home/runner/work/Slimefun4/Slimefun4/src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java:[592,48] incompatible types: org.bukkit.inventory.ItemStack cannot be converted to io.github.thebusybiscuit.slimefun4.api.items.SlimefunItemStack.
i will just look into more codes...

Outfit8TSB committed:
Update ResearchSetup.java - Fix Errors

remove Slimefunitems.COPPER_INGOT. 
fixes 
Error:  /home/runner/work/Slimefun4/Slimefun4/src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/ResearchSetup.java:[70,71] cannot find symbol
Error:    symbol:   variable COPPER_INGOT
Error:    location: class io.github.thebusybiscuit.slimefun4.implementation.SlimefunItems

Outfit8TSB committed:
Update ElectricPress.java

Outfit8TSB committed:
Merge branch 'Slimefun:experimental' into replace-sfcopper-with-mccopper

Outfit8TSB committed:
Update SlimefunItemSetup.java

Outfit8TSB committed:
Update SlimefunItemSetup.java

Outfit8TSB committed:
make Copper Dust Smelt into Minecraft Copper.

wonder it will work.
